app/templates/reward-page.html

```diff
@@ -9,7 +9,7 @@
     </div>
 
     <div ng-hide="reward.error">
-        <div class="content" ng-class="{active: reward.status == 'incomplete' || reward.status == 'pending' || reward.status == 'unverified' || reward.status == 'ineligible' || reward.status == 'updated_too_recently'}">
+        <div class="content" ng-class="{active: reward.status == 'incomplete' || reward.status == 'pending' || reward.status == 'unverified' || reward.status == 'ineligible' || reward.status == 'updated_too_recently' || reward.status == 'fb_email_missing'}">
```

Comment:
We definitely need to open an issue to look into this. This portion of the code base is way too complex. @bartekn is redesigning the rewards API and I expect most of this rewards code to get scrapped in the modular client, so it may not be worth the effort.